cesql/cesql_tck/like_expression.yaml

```diff
@@ -116,3 +116,10 @@ tests:
   - name: With type coercion from bool (4)
     expression: "FALSE LIKE 'fal%'"
     result: true 
+
+  - name: Invalid string literal in comparison causes parse error
+    expression: "x LIKE 123"
```

Comment:
The doesn't say that % is required, so why is this an error?

Reply:
Because the only valid string literals in CESQL are either single quoted or double quoted. So, since 123 is not quoted it is not a string literal. See the definition for LIKE here:
```ebnf
string-literal ::= ( "'" ( [^'] | "\'" )* "'" ) | ( '"' ( [^"] | '\"' )* '"')
like-operator ::= "LIKE"
not-operator ::= "NOT"
like-operation ::= expression not-operator? like-operator string-literal
```

Basically, the `LIKE` operator is ONLY defined where the right hand operand is a string literal, and string literals require a single or double quote